### ISSUE TITLE
Monkey_patch.py ascii encoding of sql in python2.7

### ISSUE THREAD

**Issue by Ayzse** [BertrandBordage/django-cachalot#50]
At monkey_patch.py:110 if ```sql``` is a ```str``` type, but the actual query contains unicode characters, an error is generated: 

```UnicodeDecodeError: 'ascii' codec can't decode byte 0xc3 in position 26821804: ordinal not in range(128)```

Fixed if this code is inserted before line 110:
```python
if not isinstance(sql, unicode):
    sql = unicode(sql, 'utf-8')

**Comment by BertrandBordage** [comment]
Fixed by 651a9bdf75422a20d69fe4452fb7015a54fd3868 and 0373f5eb605e51b487d6d538ff0c5bf699c9b166.

**Comment by BertrandBordage** [comment]
Note that this is not totally a bug: the type of data you can pass to `cursor.execute` is never specified in [PEP 249](https://www.python.org/dev/peps/pep-0249/) and `bytes` is not a valid type for this method with SQLite in Python 3.
